### DISCUSSION TITLE
ninja_runtests failing

### DISCUSSION THREAD

**Discussion by abhi-dev-engg** [open-quantum-safe/liboqs#1741] [General]
Hi all,

I am trying to build and run the liboqs on windows. i am successfully able to compile it on GNU
But when i run tests, some of them fail.

**_========================================================================== 54 failed, 218 passed, 313 skipped in 46.37s ==========================================================================_**

Failure reason is mentioned as below
**_FAILED tests/test_alg_info.py::test_alg_info_sig[ML-DSA-44] - FileNotFoundError: [WinError 2] The system cannot find the file specified_**

I guess the failure reason is line below
`    datasheet_filename = helpers.run_subprocess(['grep', '-r', '-l', kem_name, 'docs/algorithms/kem']).splitlines()[0]`
as there exist no file with name of ML-DSA-44 in docs/algorithms/kem.

Are there any other way to run tests ? Am i doing something wrong ?
Any help would be appreciated.

Note: I am running tests without any changes in configuration, directly on master.

Thanks.

**Comment by baentsch** [comment]
Thanks for the bug report. I have a hunch that this is a result of us not treating this algorithm as a "first class citizen"... @bhess : What'd you think? Also moving this as an issue to the proper project.

**Reply by baentsch** [reply]
Move didn't work. Now see https://github.com/open-quantum-safe/liboqs/issues/1742
